.github/workflows/test.yml

```diff
@@ -16,22 +16,9 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: 1.60.0
+          toolchain: 1.71.1
           target: wasm32-unknown-unknown
           override: true
 
       - name: Run unit tests
-        uses: actions-rs/cargo@v1
-        with:
-          command: unit-test
-          args: --locked
-        env:
-          RUST_BACKTRACE: 1
-
-      - name: Compile WASM contract
-        uses: actions-rs/cargo@v1
-        with:
-          command: wasm
-          args: --locked
-        env:
-          RUSTFLAGS: "-C link-arg=-s"
+        run: make test
```

Comment:
Don't forget to install `cargo-nextest`.

```
      - name: Install cargo-nextest
        uses: actions-rs/cargo@v1
        with:
          command: install
          args: cargo-nextest
```

Or use `cargo test` in the `test` rule in `Makefile`.